sentry-resque/Gemfile

```diff
@@ -4,12 +4,6 @@ git_source(:github) { |name| "https://github.com/#{name}.git" }
 # Specify your gem's dependencies in sentry-ruby.gemspec
 gemspec
 gem "sentry-ruby", path: "../sentry-ruby"
-gem "sentry-rails", path: "../sentry-rails"
-
-gem "rails"
-
-# For https://github.com/ruby/psych/issues/655
-gem "psych", "5.1.0"
```

Comment:
This makes tests [pass](https://github.com/getsentry/sentry-ruby/actions/runs/7942303945/job/21685467789) on Ruby 2.4 ([fail log before this change](https://github.com/getsentry/sentry-ruby/actions/runs/7942214345/job/21685304613#step:7:55))